src/components/AddAisle.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { patchItem, removeAislePrompt } from '../actions/items';
+export class AddAisle extends Component {
+  updateAisle(e) {
+    e.preventDefault();
+    const { dispatch, listId, item } = this.props;
+    item.aisleLocation = this.input.value;
+    this.input.value = '';
+    if (item.aisleLocation) {
+      dispatch(patchItem(item, listId));
```

Comment:
Unfortunately, this actually unchecks the item in the process (just tested it out).

You should instead patch it with only the fields you need to update:

```javascript
dispatch(patchItem({id: item.id, aisleLocation: item.aisleLocation}));
```

Or similar.